bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -560,7 +564,10 @@ class VideoPreview extends Component {
                 : (
                   <video
                     id="preview"
-                    className={styles.preview}
+                    className={cx({
+                      [styles.preview]: true,
```

Comment:
The true isn't needed if the class will not be used to conditional

Reply:
Perhaps, just following the convention used throughout the component file to ensure consistency.